### DISCUSSION TITLE
Recupero ID account di pagamento già esistente

### DISCUSSION THREAD

**Discussion by SupervisoreOP** [fattureincloud/api#230] [Q&A] [answered]
Buongiorno,
chiedo supporto per recuperare l'id di un account di pagamento già esistente.

La call Get Payment Account  chiede l'ID dell'account di pagamento oltre che l'id dell'azienda.

GET /c/{company_id}/settings/payment_accounts/**{payment_account_id}**

Ma come risalgo semplicemente all'id dell'account di pagamento già esistente?

Grazie.

**Comment by valmoz** [comment, marked as answer]
Ciao @SupervisoreOP 
puoi usare il metodo [ListPaymentAccounts](https://developers.fattureincloud.it/api-reference#get-/c/-company_id-/info/payment_accounts)

Mauro
